client/app/students/students.html

```diff
@@ -61,19 +61,23 @@
                 </tr>
                 </thead>
                 <tbody md-body>
-                <tr md-row md-select="item" md-select-id="id" md-auto-select ng-repeat="item in items.docs">
+                <tr md-row ng-attr-md-row-select="isEditor()" md-select="item" md-select-id="id" md-auto-select  ng-repeat="item in items.docs" >
                     <td md-cell>{{ item.id }}</td>
-                    <td md-cell ng-click="edit($event, item, 'name.first')">{{ item.name.first }}</td>
-                    <td md-cell ng-click="edit($event, item, 'name.last')">{{ item.name.last }}</td>
-                    <td md-cell ng-click="edit($event, item, 'class')">{{ item.class }}</td>
-                    <td md-cell>
-                        <md-input-container>
-                            <md-select ng-model="item.gender" class="md-no-underline" ng-change="changeGender(item)">
+                    <td md-cell ng-if="isEditor()" ng-click="edit($event, item, 'name.first')">{{ item.name.first }}</td>
+                    <td md-cell  ng-if="!isEditor()">{{ item.name.first }}</td>
+                    <td md-cell ng-if="isEditor()" ng-click="edit($event, item, 'name.last')">{{ item.name.last }}</td>
+                    <td md-cell  ng-if="!isEditor()">{{ item.name.last }}</td>
+                    <td md-cell ng-if="isEditor()" ng-click="edit($event, item, 'class')">{{ item.class }}</td>
+                    <td md-cell ng-if="!isEditor()">{{ item.class }}</td>
+                    <td md-cell ng-if="isEditor()">
+                        <md-input-container >
+                            <md-select ng-model="item.gender" class="md-no-underline" ng-change="changeGender(item)" >
                                 <md-option value="female">Female</md-option>
                                 <md-option value="male">Male</md-option>
                             </md-select>
                         </md-input-container>
                     </td>
+                    <td md-cell ng-if="!isEditor()">{{ item.gender }}</td>
```

Comment:
Here we could use ng-disabled on the md-select instead of ng-if for the whole td